package.json

```diff
@@ -58,27 +59,23 @@
     "node": ">=0.10"
   },
   "dependencies": {
-    "assert-plus": "^0.1.5",
-    "backoff": "^2.4.0",
+    "assert-plus": "0.1.5",
```

Comment:
FWIW, I don't think carets work on 0.x releases anyway, cause of some escape clause thingy with 0.x (blame semver).